Makefile

```diff
@@ -126,6 +126,9 @@ deploy-app: ## Deploys the app to PaaS
 	# Delete the route for the old app
 	@if cf app ${APPLICATION_NAME} >/dev/null; then ./scripts/unmap-route.sh ${APPLICATION_NAME}; fi
 
+	# Make sure relevant routes are whitelisted (can be found in vars/ip_whitelist_routes.json)
+	.scripts/add-ip-whitelist-route-service.sh ${APPLICATION_NAME}
```

Comment:
Should the new route be whitelisted before we unmap the old route?

Reply:
I _think_ it will be fine to do it before we unmap actually yeah. I've updated and I'm going to give it a whirl on preview